backend/src/main/scala/cromwell/backend/standard/callcaching/BlacklistCache.scala

```diff
@@ -2,23 +2,59 @@ package cromwell.backend.standard.callcaching
 
 import com.google.common.cache.{CacheBuilder, CacheLoader}
 import cromwell.core.CacheConfig
+import cromwell.services.CallCaching.CallCachingEntryId
 
-case class BlacklistCache(config: CacheConfig) {
-  val cache = {
-    // Queries to the blacklist cache return false by default (i.e. not blacklisted).
-    val falseLoader = new CacheLoader[String, java.lang.Boolean]() {
-      override def load(key: String): java.lang.Boolean = false
+sealed trait BlacklistStatus
+case object KnownBad extends BlacklistStatus
+case object KnownGood extends BlacklistStatus
+case object Unknown extends BlacklistStatus
```

Comment:
Could these be a little more descriptive? eg `GoodCacheResult`, `BadCacheResult`, `UntestedCacheResult`?